.github/workflows/push.yml

```diff
@@ -0,0 +1,33 @@
+name: Bump version
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  bump-version:
+    name: "Bump Version on master"
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: "Checkout source code"
+        uses: "actions/checkout@v2"
+        with:
+          ref: ${{ github.ref }}
+          fetch-depth: 0
+      - name: "fetch default branch"
+        run: git fetch origin ${{ github.repository.default_branch }}
+      - name: "cat package.json"
+        run: cat ./package.json
+      - name: "Setup Node.js"
+        uses: "actions/setup-node@v1"
+        with:
+          node-version: 8.10.0
```

Comment:
since the docker image has node already, I feel like this is not necessary

Reply:
ohh, true!